qa/rpc-tests/wallet_anchorfork.py.original

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python2
+# Copyright (c) 2016 The Zcash developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import *
+from time import *
+
+class WalletAnchorForkTest (BitcoinTestFramework):
```

Comment:
This test doesn't seem to be related to the subject of the PR.

Reply:
force-pushed fix